src/OpenColorIO/PathUtils.cpp

```diff
@@ -36,7 +34,8 @@ namespace
 // https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?redirectedfrom=MSDN&view=vs-2019
 // "The inode, and therefore st_ino, has no meaning in the FAT, HPFS, or NTFS file systems."
 
-std::string ComputeHash(const std::string & filename)
+// Default Compute Hash Function
```

Comment:
The convention is to have sentences for explanations.
// That's the default hash method implementation to compute a hash key based on a file content.